gspconsumer/nitghtime.py

```diff
@@ -0,0 +1,74 @@
+""" Add zeros to nightime """
+import logging
+import os
+from datetime import datetime
+
+import pandas as pd
+import pvlib
+from nowcasting_datamodel.models.gsp import LocationSQL
+
+logger = logging.getLogger(__name__)
+
+elevation_limit = os.getenv("ELEVATION_LIMIT", 5)
```

Comment:
Comment explaining this magic number wouldn't go amiss!

Reply:
thanks